docs/contributing/application-architecture.md

```diff
@@ -43,4 +43,4 @@
 
 - `src/`
 
-  Source files. See README.md in the src directory for details.
+  Source files. See [README.md](/src/govuk/README.md) in the src directory for details.
```

Comment:
Looks like the README got moved from `src/` to `src/govuk` but I've linked it up anyway so we can keep track once directories move